src/background/Wallet/Wallet.ts

```diff
@@ -1049,7 +1077,7 @@ export class Wallet {
     }
   }
 
-  private getOfflineSigner() {
+  getOfflineSigner() {
```

Comment:
Why can't we use `wallet.personalSign` method from wallet instead of exporting signer?

Reply:
Yes, `personalSign` does exactly what we need here, I don't already remember what was my initial idea when I decided to get a signer

Reply:
Actually this is wrong. I found an error that you discovered Iskander reported today: we're signing this message using the current wallet address as a signer. But we should use the newly created address to sign this "refer wallet" message